build_info/ddrescue.control

```diff
@@ -0,0 +1,40 @@
+Package: ddrescue
```

Comment:
I'd actually prefer this package name to be up with Debian (``gddrescue``), since I've had this issue where I can't figure what to install since packages we have have different names.

You should also rename your Makefile and control file to reflect this name as well.

Reply:
Well considering (`ddrescue` was removed from Debian because of confusion between it and GNU `ddrescue`)[https://tracker.debian.org/news/444590/bug677101-removed-packages-from-unstable/] I think it’s safe to name to differently from Debian.

Reply:
Most packages have always maintained their names based on what Debian names them so that there's no confusion, and we should try to avoid that to begin with.